### PR TITLE
date-picker-token

### PR DESCRIPTION
**Pull request by Ddungyov**
adding them

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -1930,6 +1930,20 @@
       "type": "typography"
     }
   },
+  "date picker": {
+    "gtmds_date-picker_margin": {
+      "value": "{gtmds_space-xxs}/2",
```
Comment (StefanIvanov) [inline data/tokens.json:1935]:
```suggestion
      "value": "{gtmds_size-sm}",
```

File: data/tokens.json [hunk 2]
```diff
@@ -1930,6 +1930,20 @@
...
+      "type": "spacing"
+    },
+    "gtmds_date-picker_padding-horizontal": {
+      "value": "{gtmds_space-sm}/2",
```
Comment (StefanIvanov) [inline data/tokens.json:1939]:
```suggestion
      "value": "{gtmds_size-lg}-{gtmds_size-sm}",
```

File: data/tokens.json [hunk 3]
```diff
@@ -1930,6 +1930,20 @@
...
+      "type": "spacing"
+    },
+    "gtmds_date-picker_padding-vertical": {
+      "value": "{gtmds_space-xxs}/2",
```
Comment (StefanIvanov) [inline data/tokens.json:1943]:
```suggestion
      "value": "{gtmds_size-sm}",
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]
We use the size tokens for paddings and margins and the space ones are kept only in case we decide to use them in composites e.g. for vertical spacers between form elements

File: data/tokens.json (hunk 3 above)
Comment (Ddungyov) [reply data/tokens.json:1943]: Still in the works, but I will update this. Good reminder.

File: data/tokens.json [hunk 4]
```diff
@@ -1930,6 +1930,89 @@
...
+    },
+    "gtmds_date-picker-option_background-color-hover": {
+      "value": "{gtmds_color_blue-1}",
+      "type": "color"
+    },
+    "gtmds_date-picker-option_background-color-range": {
+      "value": "{gtmds_color_blue-2}",
+      "type": "color"
+    },
+    "gtmds_date-picker-option_background-color-range-hover": {
```
Comment (StefanIvanov) [inline data/tokens.json:1958]:
```suggestion
    "gtmds_date-picker-option-range_background-color-hover": {
```

File: data/tokens.json [hunk 5]
```diff
@@ -1930,6 +1930,89 @@
...
+      "type": "spacing"
+    },
+    "gtmds_date-picker-option_height": {
+      "value": "{gtmds_size-xl}+{gtmds_size-lg}-{gtmds_size-sm}",
+      "type": "sizing"
+    },
+    "gtmds_date-picker-option_background-color-hover": {
+      "value": "{gtmds_color_blue-1}",
+      "type": "color"
+    },
```
[the remark this answers is not in the document]
Comment (StefanIvanov) [reply data/tokens.json:1953]: you forgot to add the normal state background color token and this suggestion will add it

**Comment by StefanIvanov** [comment]
@Ddungyov you can go through my suggested changes and accept them if you agree right here, no need to go to Figma, make changes, commit and push ;)

**Pull request merged**